components/crash/core/common/crash_key.h

```diff
@@ -20,7 +20,7 @@
 // Annotation interface. Because not all platforms use Crashpad yet, a
 // source-compatible interface is provided on top of the older Breakpad
 // storage mechanism.
-#if BUILDFLAG(USE_CRASHPAD_ANNOTATION)
+#if BUILDFLAG(USE_CRASHPAD_ANNOTATION) && !defined(STARBOARD)
```

Comment:
We have crashpad, is there a reason why these should be all disabled ? Should be documented somewhere. CC @andrewsavage1

Reply:
So all the crashpad stuff is for things like reporting crash metrics and hooks that upload logs after/before a crash. While it might be possible to do this integration in Cobalt, as a first pass it was easier to leave it disabled. There was a lot of Chrome-specific crashpad integration stuff that would need to be thought through for application here. I've filed b/286881972 and added a TODO here to link it.